knode/backend.go

```diff
@@ -336,7 +344,7 @@ func (s *Kowala) StartValidating() error {
 	}
 
 	deposit, err := s.Deposit()
-	if err != nil {
+	if err != nil && err != validator.ErrIsNotRunning {
```

Comment:
I think the start as a deferred, so not sure why this check

Reply:
This is needed to not to return `validator.ErrIsNotRunning` in `validator.start()`.